apps/frontend/src/components/global/sidebaritems/SidebarFileList.vue

```diff
@@ -91,15 +89,14 @@ export default class FileItem extends mixins(ServerMixin) {
   save_evaluation(file: EvaluationFile) {
     this.saving = true;
 
-    let evaluationDTO: ICreateEvaluation = {
-      data: file.evaluation.data,
-      filename: file.filename,
-      userId: ServerModule.userInfo.id,
-      evaluationTags: []
-    };
```

Comment:
Does this remove the ability to create tags?

Reply:
The tag functionality shouldn't need to live in inspecjs at all, for the way heimdall2 is setup it doesn't really make sense to add tags on save, however for someone uploading from their own CI job they would probably want the ability to add tags and groups as part of the save.